runtime/gc_glue_java/MarkingDelegate.cpp

```diff
@@ -272,7 +272,8 @@ MM_MarkingDelegate::scanContinuationNativeSlots(MM_EnvironmentBase *env, omrobje
 		bStackFrameClassWalkNeeded = isDynamicClassUnloadingEnabled();
 #endif /* J9VM_GC_DYNAMIC_CLASS_UNLOADING */
 
-		GC_VMThreadStackSlotIterator::scanSlots(currentThread, objectPtr, (void *)&localData, stackSlotIteratorForMarkingDelegate, bStackFrameClassWalkNeeded, false);
+		bool syncWithConcurrentGCScan = _extensions->shouldScavengeNotifyGlobalGCOfOldToOldReference();
```

Comment:
Simplest is to rely on J9_PRIVATE_FLAGS_CONCURRENT_MARK_ACTIVE
It was probably meant to be used by mutator threads, but we set it on any thread (including GC ones)